### PR TITLE
Added 'shopping' protocol that allows to qucikly sell all alcohol items.

### PR DESCRIPTION
**Pull request by VectorAdw**
## New Protocol
```
BD.Shopper.SellAlcohol()
```

### CODE REVIEW

File: mods/braindance_protocol/utility/shopper.lua [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "Items.TopQualityAlcohol4", -- Sake Utagawa
+    "Items.TopQualityAlcohol5", -- Baalbek Arak
+    "Items.TopQualityAlcohol6", -- Romvlvs Gin
+    "Items.TopQualityAlcohol7" -- Paul Night
+}
+
+function Shopper.SellAlcohol()
+    -- Initialize protocol
+    local moduleName = "Mercenaries Anonymous"
+    Utilities.StartProtocol(moduleName)
```
Comment (osulli) [inline mods/braindance_protocol/utility/shopper.lua:50]: Excellent 😆

File: mods/braindance_protocol/utility/shopper.lua [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    local totalAlcoholCount = 0
+    for index, alcohol in ipairs(alcohols) do
+		local alcoholTID = TweakDBID.new(alcohol)
+        local alcoholID = ItemID.new(alcoholTID)
+        local currentAlcoholCount = transactionSystem:GetItemQuantity(player, alcoholID)
+        transactionSystem:RemoveItem(player, alcoholID, currentAlcoholCount)
+        totalAlcoholCount = totalAlcoholCount + currentAlcoholCount
+    end
+
+    -- Add Eddies - alcohol value is constant for currently available items
```
Comment (osulli) [inline mods/braindance_protocol/utility/shopper.lua:66]: Good to know.

**Review by osulli — APPROVED** [review]
Looks great.
Can't test personally, but if you're happy then please merge.

Great work and thanks for contributing to a community repository :)

File: mods/braindance_protocol/utility/shopper.lua (hunk 2 above)
Comment (VectorAdw) [reply mods/braindance_protocol/utility/shopper.lua:66]: Yup, all of them worth 30 eddies, even the unused generics.
Looks like someone forgot to adjust base value ¯\\_(ツ)_/¯.

**Pull request merged**